tee-worker/litentry/core/assertion-build/src/brc20/amount_holder.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020-2023 Trust Computing GmbH.
+// This file is part of Litentry.
+//
+// Litentry is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Litentry is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Litentry.  If not, see <https://www.gnu.org/licenses/>.
+
+#[cfg(all(feature = "std", feature = "sgx"))]
+compile_error!("feature \"std\" and feature \"sgx\" cannot be enabled at the same time");
+
+#[cfg(all(not(feature = "std"), feature = "sgx"))]
+extern crate sgx_tstd as std;
+
+use crate::*;
+use lc_credentials::brc20::amount_holder::BRC20AmountHolderCredential;
+use lc_data_providers::{
+	geniidata::GeniidataClient, DataProviderConfigReader, ReadDataProviderConfig,
+};
+
+pub fn build(req: &AssertionBuildRequest) -> Result<Credential> {
+	let identities = transpose_identity(&req.identities);
+
+	// TODO: here still missing one step: convert 'address', which is a pubkey, into real BTC/BRC20 address,
+	// which the data provider (GeniiData.com) can recognize.
+	let addresses = identities
```

Comment:
`GeniiData` doesn't support pubkey?
So what format of address should be converted here？